packages/core/test/CubeResource.js

```diff
@@ -11,4 +17,11 @@ describe('PIXI.resources.CubeResource', function ()
             new CubeResource(8);
         }).to.throw(Error, /invalid length/i);
     });
+    it('should be created through BaseTexture.from()', function ()
+    {
+        const path1 = this.baseTexUrl;
+        const baseTex = BaseTexture.from([path1, path1, path1, path1, path1, path1]);
+
+        expect(baseTex.resource instanceof CubeResource).to.be.true;
```

Comment:
what about `expect(baseTex.resource).to.be.instanceof(CubeResource)`?

Reply:
Yes, better!